fdbserver/ClusterController.actor.cpp

```diff
@@ -1498,7 +1500,9 @@ ACTOR Future<Void> statusServer(FutureStream<StatusRequest> requests,
 			                                                                  coordinators,
 			                                                                  incompatibleConnections,
 			                                                                  self->datacenterVersionDifference,
-			                                                                  configBroadcaster)));
+			                                                                  configBroadcaster,
+			                                                                  self->db.metaclusterRegistration,
+			                                                                  self->db.metaclusterMetrics)));
```

Comment:
There is also the alternative of passing in the `ClusterControllerData` itself and not having an extended list of parameters

Reply:
I considered that too but left it as is because it would make a lot of the existing code more packed and harder to read because of needing to dereference an extra `data->db`